lilypad/cli/commands/local.py

```diff
@@ -33,8 +32,19 @@ def _start_lilypad(project_dir: Path, port: str | int) -> subprocess.Popen:
 
     # Sets the LILYPAD_PROJECT_DIR so Lilypad knows where to look for the database
     env["LILYPAD_PROJECT_DIR"] = str(project_dir)
-    env["LILYPAD_LOCAL"] = "true"
+    env["LILYPAD_ENVIRONMENT"] = "local"
     files_dir = files("lilypad").joinpath("server")
+    if not os.path.exists("pad.db"):
+        subprocess.run(
+            [
+                "python",
+                "db/setup.py",
```

Comment:
will this run properly in a user's environment when they install lilypad? previously we just used the `create_tables` method, was there something wrong with that?

Reply:
i can test in a fresh environment. the issue with the previous method was that i had no way of knowing which environment they were in. so with the new method, we set the environment to local, instead of it using the default which is dev.

Reply:
could we update `create_tables` to take the environment as an argument?